common.yml

```diff
@@ -13,6 +13,8 @@ mongo_tools_variables:
   mongo_tools_task_lists:
     mac_task_list: &macos_tasks
       - name: dist
+      - name: sign
+        run_on: amazon1-2018-test
```

Comment:
Not every evg distro has the notary client on it, which is why we can't just bundle signing into the dist task. This is one that happens to have it available, so we will run every sign task on it, regardless of what the default distro for the buildvariant is.

Reply:
I say, let's add this comment to the file. It'll help people looking at it in the future :)